src/tapatar.js

```diff
@@ -180,6 +187,16 @@
 
             return $el;
         },
+        _addCropper: function(source) {
+            var $img = this.$containerEl.find('.tptr-big-image');
+            $img
+                .cropper(this.options.cropper.options)
+                .on('crop.cropper', this._cropEvent.bind(this, source, $img));
+        },
+        _cropEvent: function(source, $img) {
+            source.setImageData($img.cropper('getCroppedCanvas').toDataURL());
+            this._setPickedImage(source);
```

Comment:
in case you wonder why I don't set the 2nd parameter to true (`setImageData`): It would end in a cropper init loop.
That is why _setPickedImage has to be called separately.